### DISCUSSION TITLE
How do I extract my world from this container? I am either going to move it to another raspi or to a dedicated server.

### DISCUSSION THREAD

**Discussion by couchpotatochip21** [itzg/docker-minecraft-bedrock-server#407] [Q&A]
I need to move my minecraft world from this container to another raspi or a dedicated server.

**Comment by itzg** [comment]
If you have your data volume mounted from a host directory, then just move that host directory. Otherwise, use `docker cp` command to transfer contents out of the anonymous or named volume.
